index.html

```diff
@@ -8,8 +8,9 @@ <h1 id="title">Jerry Lawson</h1>
   <body id="body">
     <main id="main">
       <div id="img-div">
-      <h6>"The Black video game engineer who pioneered the way we play."</h6>
-      <img id="image" src="https://thecurrent-online.com/wp-content/uploads/2022/12/Jerry_lawson_ca_1980.png" alt="Jerry Lawson pic">
+        <h6>"The Black video game engineer who pioneered the way we play."</h6>
+        <img id="image" src="https://thecurrent-online.com/wp-content/uploads/2022/12/Jerry_lawson_ca_1980.png" alt="Jerry Lawson pic">
+      </div>
```

Comment:
I added a closing `</div>` tag here to make sure we close the `<div>` on line 10.